docs/docs/other/faq.md

```diff
@@ -4,29 +4,43 @@ title: Frequently Asked Questions
+### Where can I download a build?
 
-We're developing Onivim 2 [in the open](https://github.com/onivim/oni2), but it is licensed under a [commercial EULA](https://github.com/onivim/oni2/blob/master/Outrun-Labs-EULA-v1.1.md).
+Pre-alpha builds are available for users who have pre-ordered via our [Early Access Portal](https://v2.onivim.io/early-access-portal).
 
-However, we value open source - and to that end, we've implemented a 'time-delay' open source license. __Each commit that makes it to `master` will be dual-licensed under the permissive MIT license after 18 months__.
-We maintain a [separate repo](https://github.com/onivim/oni2-mit) containing the MIT licensed code, which is sync'd daily. 
+If you haven't pre-ordered, you can still try it out by following our [build instructions](../for-developers/building).
 
-We hope that this can help us strike a balance: __the sustainability of a commercial offering__ while __giving back to the open source community__.
+### How do I update my build?
 
-### Where can I download a build?
+Nightly builds are published to the [Early Access Portal](https://v2.onivim.io/early-access-portal).
 
-Pre-alpha builds are available for users who have pre-ordered via our [Early Access Portal](https://v2.onivim.io/early-access-portal).
+Auto-update is a [work-in-progress](https://github.com/onivim/oni2/issues/559), until it is implemented, you'll need to download the latest build manually from the [Early Access Portal](https://v2.onvim.io/early-access-portal).
 
-If you haven't pre-ordered, you can still try it out by following our [build instructions](../for-developers/building).
+## Licensing
+
+### Where is my license key?
+
+After purchase, you'll receive an invoice from `paypal@outrunlabs.com` with your key attached.
+
+If you don't receive it - please reach out to [hello@outrunlabs.com](mailto:hello@outrunlabs.com) and we'll get it set for you.
 
 ### I already backed Onivim 1, do I need to pay again?
 
 __Nope!__ If you supported the project via any of the crowd-sourcing platforms prior to July 31, 2019, you have a __lifetime commercial license__.
 
 If you did not receive a license key - feel free to send me a mail at bryphe@outrunlabs.com and we'll get you set. THANK YOU for supporting the project!
 
+### Is Onivim 2 open-source?
+
+We're developing Onivim 2 [in the open](https://github.com/onivim/oni2), but it is licensed under a [commercial EULA](https://github.com/onivim/oni2/blob/master/Outrun-Labs-EULA-v1.1.md).
+
+However, we value open source - and to that end, we've implemented a 'time-delay' open source license. __Each commit that makes it to `master` will be dual-licensed under the permissive MIT license after 18 months__.
+We maintain a [separate repo](https://github.com/onivim/oni2-mit) containing the MIT licensed code, which is sync'd daily. 
+
+We hope that this can help us strike a balance: __the sustainability of a commercial offering__ while __giving back to the open source community__. 
```

Comment:
I'd probably also reference here that a portion of the money Oni2 gets also goes back into Open Source projects that power it.

Reply:
Good point!

Reply:
We haven't really publicized the funding we push back to Vim/Neovim/others that much...  would be worth mentioning.